docs/source/cheat_sheet.rst

```diff
@@ -0,0 +1,302 @@
+|                          |       >>> obs_regrid = obs.regridc([time], axes='T', method='linear')                                                                                                |
++--------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|      **Aggregation**     | *Aggregate all northward wind fields into a new construct:*                                                                                                          |
+|                          |                                                                                                                                                                      |
+|                          | .. code-block:: python                                                                                                                                               |
+|                          |                                                                                                                                                                      |
+|                          |       >>> v = cf.read("~/northward_wind*.nc")[0]                                                                                                                     |
+|                          |       >>> v_all = cf.aggregate(v)                                                                                                                                    |
+|                          |                                                                                                                                                                      |
+|                          | *Aggregate allowing for missing coordinate identities to be inferred:*                                                                                               |
+|                          |                                                                                                                                                                      |
+|                          | .. code-block:: python                                                                                                                                               |
+|                          |                                                                                                                                                                      |
+|                          |       >>> f = cf.read("~/realization/PRMSL.1941_mem*.nc")[0:50]                                                                                                      |
+|                          |       >>> ensemble = cf.aggregate(f, dimension=("realization",), relaxed_identities=True)[0]                                                                         |
```

Comment:
```suggestion
```
As discussed offline, we'll remove this for now, and decide how best to represent aggregation later.